test/Algebra/Graph/Test/RewriteRules.hs

```diff
@@ -111,16 +105,15 @@ inspect $ 'bind2 === 'bind2R
 
 -- Ideally, we want this test to pass.
 -- Strangely, '<*>' in 'ovApR' does not inline and makes the test fail.
---
--- This is corrected below, where '<*>' was inlined "by hand"
 ovAp, ovApR :: Graph (a -> b) -> Graph (a -> b) -> Graph a -> Graph b
 ovAp  x y z = overlay x y <*> z
 ovApR x y z = overlay (x <*> z) (y <*> z)
 
 inspect $ 'ovAp =/= 'ovApR
 
+-- Here we inline "by hand", but we see different results on GHC 8.6 vs GHC 8.8!
 ovAp', ovApR' :: Graph (a -> b) -> Graph (a -> b) -> Graph a -> Graph b
 ovAp'  x y z = overlay x y <*> z
 ovApR' x y z = overlay (x >>= (<$> z)) (y >>= (<$> z))
 
-inspect $ 'ovAp' === 'ovApR'
+-- inspect $ 'ovAp' =/= 'ovApR'
```

Comment:
Perhaps, instead of commenting this out, you could use `CPP` to use `===` or `=/=` depending on the compiler version?

Reply:
Reasonable! I'll take care of that.